Kaplekar Constant/Assignment2.cpp

```diff
@@ -0,0 +1,69 @@
+#include <iostream>
+using namespace std;
+
+bool areDigitsSame(int number) {
+    int digit = number % 10;
+    while (number > 0) {
+        if (number % 10 != digit) {
+            return false;
+        }
+        number /= 10;
+    }
+    return true;
+}
+
+void sorting(int arr[], int size) {
+      for (int i = 0; i < size - 1; i++) {
+        for (int j = 0; j < 3 - i; j++) {
+            if (arr[j] > arr[j + 1]) {
+                int temp = arr[j];
+                arr[j] = arr[j + 1];
+                arr[j + 1] = temp;
+            }
+        }
+    }
+}
+  
+int kaprekar(int number) {
```

Comment:
Function can be a verb which define action so **Kaprekar** is not a good naming convention.